### ISSUE TITLE
Install error on Eagle

### ISSUE THREAD

**Issue by elainethale** [NREL/jade#29]
After creating a conda environment in a bash file with

```bash
conda create -y --prefix "$MY_CONDA_PREFIX" -c conda-forge "pyarrow>=0.14" python=3.8 pandas 
```

and then running

```bash
source activate "$MY_CONDA_PREFIX"
```

```
"$MY_CONDA_PREFIX/bin/pip" install git+ssh://git@github.com/NREL/jade.git@master
```

failed with the error:

```bash
Building wheels for collected packages: jade
  Building wheel for jade (PEP 517) ... done
  Created wheel for jade: filename=jade-number-py3-none-any.whl size=114672 sha256=ddccba07aa1db7671c99784e225d4ea88c8115b2113d81514ebce2e8050af0ee
  Stored in directory: /tmp/pip-ephem-wheel-cache-sbm3pkdc/wheels/d6/9e/64/c80fa3dc911ba0a566d6d10df981758756998f916ee6e9876d
  WARNING: Built wheel for jade is invalid: Metadata 1.2 mandates PEP 440 version, but 'number' is not
Failed to build jade
ERROR: Could not build wheels for jade which use PEP 517 and cannot be installed directly
```

**Comment by elainethale** [comment]
```bash
pip install git+ssh://git@github.com/NREL/jade.git@v0.2.9
```
gave the same error:

```
Building wheels for collected packages: jade
  Building wheel for jade (PEP 517) ... done
  Created wheel for jade: filename=jade-number-py3-none-any.whl size=114602 sha256=ab2cd57cbb2e0ff80df4df81a447bf0e820eb492a4f39be434a601103ba9a67f
  Stored in directory: /tmp/pip-ephem-wheel-cache-jqwa3fje/wheels/12/f2/48/57c45f6357e101c1b54f91fd454851e440224f091b7cf5131c
  WARNING: Built wheel for jade is invalid: Metadata 1.2 mandates PEP 440 version, but 'number' is not
Failed to build jade
ERROR: Could not build wheels for jade which use PEP 517 and cannot be installed directly
```

**Comment by elainethale** [comment]
Going back to v0.1.3 works--it installs with a warning. I do not know why this older version succeeds but the newest ones do not:

```
Building wheels for collected packages: jade
  Building wheel for jade (setup.py) ... done
  Created wheel for jade: filename=jade-number-py3-none-any.whl size=91158 sha256=05680fa37981a686a60d6c65374c05f1f2c04ed85cffdf3095a7ecb62acd11c7
  Stored in directory: /tmp/pip-ephem-wheel-cache-5onw1e37/wheels/a3/c5/ee/14e1b06fb6bf1362e947da1b811d8164049568c26ead0bcb9d
  WARNING: Built wheel for jade is invalid: Metadata 1.2 mandates PEP 440 version, but 'number' is not
Failed to build jade
Installing collected packages: scipy, patsy, numexpr, kiwisolver, cycler, toml, tables, statsmodels, pyyaml, psutil, prettytable, matplotlib, filelock, feather-format, click, jade
    Running setup.py install for jade ... done
  DEPRECATION: jade was installed using the legacy 'setup.py install' method, because a wheel could not be built for it. A possible replacement is to fix the wheel build issue reported above. You can find discussion regarding this at https://github.com/pypa/pip/issues/8368.
```

**Comment by elainethale** [comment]
Installing v0.2.8 works for now, with the same warning as v0.1.3

**Comment by T-Man-Stan** [comment]
@elainethale did you try using just 'conda activate' in the .sh file? conda activate causes me errors but perhaps the opposite is occurring here where 'source activate' is causing you errors? I was able to install JADE fine in my new environment with the "$MY_CONDA_PREFIX/bin/pip" install git+ssh://git@github.com/NREL/jade.git@master" line in the .sh file.

**Comment by elainethale** [comment]
I didn't hit the "Unable to activate" error in our install script, and in any case, my additional tests with the other versions were just done on the command line, after the environment was activated. The error was replicated there, in addition to showing up when I ran our install script.

**Comment by daniel-thom** [comment]
I'm fairly clueless on Python package management and not sure what is wrong yet. This worked for me. I'll look for a proper fix.
```
pip install git+https://github.com/NREL/jade.git@v0.2.10 --no-use-pep517 jade
```

**Comment by daniel-thom** [comment]
This problem was introduced in v0.2.9 because I added `pyproject.toml` (in order to use the `black` formatting tool) without specifying the build system correctly. I removed the file until I have time to figure out how to do it properly. Now, `pip install git+https://github.com/NREL/jade.git@v0.2.11` will work, but will give a warning message and a deprecation message. I'll leave the issue open until I properly fix it.

Sorry for the trouble.
